services/search/utils.py

```diff
@@ -1,7 +1,38 @@
-import requests  # pylint: disable=import-error
+import logging
+from requests_futures.sessions import FuturesSession  # pylint: disable=import-error
 import constants
 
 
+class SynonymExtractor:
+    """
+    Wrapper class for requests_futures methods
+    """
+    def __init__(self, max_synonyms):
+        self.max_synonyms = max_synonyms
+        self.session = FuturesSession()
+
+    def create_conc_reqs(self, word_arr):
```

Comment:
nit: it's an implementation detail that the requests are concurrent, so it would be more appropriate to name this method `create_reqs`